pytest_splunk_addon/standard_lib/data_models/Web.json

```diff
@@ -169,24 +189,9 @@
         },
         {
           "name": "http_content_type",
-          "type": "optional",
+          "type": "required",
```

Comment:
again missing about as often as it's there https://github.com/search?q=org%3Asplunk++%3Cfield%3Ehttp_content_type&type=code

https://github.com/search?q=org%3Asplunk++%3Cfield+name%3D%22http_content_type%22+value%3D%22&type=code

Reply:
Please revert as **optional**, i corrected it in the sheet, my bad.

Reply:
done